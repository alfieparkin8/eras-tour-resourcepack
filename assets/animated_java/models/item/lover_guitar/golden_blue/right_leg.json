{
	"parent": "animated_java:item/lover_guitar/right_leg",
	"textures": {
		"1": "animated_java:item/lover_guitar/golden_layer_1",
		"2": "animated_java:item/lover_guitar/lover_blue",
		"5": "animated_java:item/lover_guitar/golden_boots_layer_1"
	}
}