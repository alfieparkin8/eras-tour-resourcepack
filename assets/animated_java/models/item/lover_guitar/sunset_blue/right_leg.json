{"parent":"animated_java:item/lover_guitar/right_leg","textures":{"1":"minecraft:block/guitars/lover_blue"}}