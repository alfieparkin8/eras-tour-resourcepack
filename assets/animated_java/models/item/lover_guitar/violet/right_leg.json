{"parent":"animated_java:item/lover_guitar/right_leg","textures":{"2":"minecraft:block/outfits/lover/violet_layer_1"}}