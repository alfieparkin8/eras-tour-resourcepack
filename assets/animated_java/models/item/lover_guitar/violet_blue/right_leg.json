{"parent":"animated_java:item/lover_guitar/right_leg","textures":{"1":"minecraft:block/guitars/lover_blue","2":"minecraft:block/outfits/lover/violet_layer_1"}}