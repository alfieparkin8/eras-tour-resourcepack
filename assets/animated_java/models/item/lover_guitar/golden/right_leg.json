{
	"parent": "animated_java:item/lover_guitar/right_leg",
	"textures": {
		"1": "animated_java:item/lover_guitar/golden_layer_1",
		"5": "animated_java:item/lover_guitar/golden_boots_layer_1"
	}
}